subsys/bootloader/Kconfig

```diff
@@ -167,7 +167,10 @@ config PM_PARTITION_SIZE_B0_IMAGE
 	default 0x8000 if !B0_MIN_PARTITION_SIZE
 	default FPROTECT_BLOCK_SIZE if SOC_NRF9160 || SOC_NRF5340_CPUAPP
 	default 0x3800 if SOC_NRF5340_CPUNET
-	default 0x7000
+    # The final default should be for devices without OTP, as the provision
+    # image will add 0x1000 to the partition size. The following value
+    # accommodates that addition to all fit within 16kB
+	default 0x2e00
```

Comment:
0x2e00 + 0x1000 = 0x3e00

What about the last 0x200 to get to 0x4000 (16k)?

Reply:
I took the `0x3e00` from [sdk-mcuboot's external crypto overlay](https://github.com/nrfconnect/sdk-mcuboot/blob/master/boot/zephyr/overlay-minimal-external-crypto.conf#L46). The `0x200` is MCUboot's pad size, but since that isn't relevant here I saw this difference as forcing it to compile less than 16kB while keeping a consistent value across bootloaders.

Using `0x4000` is fine with me though, as it's more relevant to b0.

Reply:
Using 0x3000 instead of 0x2e00 makes much more sense. Updated the value.